.README/rules/tag-lines.md

```diff
@@ -21,9 +21,13 @@ Use with "always" to indicate the number of lines to require be present.
 Use with "always" to indicate the normal lines to be added after tags should
 not be added after the final tag.
 
-##### `dropEndLines` (defaults to `false`)
+##### `startLines` (defaults to `undefined`)
 
-If defined, will drop end lines for the final tag only.
+If defined, will enforce end lines to the given count before the first tag only.
+
+##### `endLines` (defaults to `undefined`)
```

Comment:
Is `noEndLines` needed now, or can this just be set to 0?

The description for `noEndLines` could probably be re-written, it reads a bit strange and if both are needed maybe mention there how it relates to `endLines`.

Reply:
`noEndLines` means that the normal behavior of the rule doesn't apply to the very last tag.

This could probably be renamed to `applyToEndTag` (with the boolean value flipped).

Although the option might be dropped in favor of disallowing application of the rule to the final tag unless users use `endLines` to enforce the count there regardless of tag, I think that is less than ideal because then users cannot decide to have different final whitespace depending on the final tag's name as they can currently.

Reply:
Tricky. A name change might be good. It is a bit tricky to get it right with all three rules `sort-tags`, `tag-lines` and `newline-after-description`. So dropping one or more options and renaming a bit might clear this confusion up.

Reply:
I also wonder if `tag-linebreaks` would also be more explanatory, but that is a minor thing and probably not worth a rename.

Reply:
Yeah, I went ahead and dropped `noEndLines: true` in favor of `applyToEndTag: false`. Hate to be making breaking changes, no less 3 breaking changes (dropping `newline-after-description` in favor of `startLines` and dropping `dropEndLines` in favor of `endLines: 0`), but I think it's all for the best.

`tag-line-breaks` would be more descriptive, but yeah, I'd rather not be doing any more breakage for such kinds of gains.